Formula/aviatorscript.rb

```diff
@@ -0,0 +1,32 @@
+class Aviatorscript < Formula
+  desc "Lightweight, high performance scripting language hosted on the JVM"
+  homepage "http://fnil.net/aviator/"
+  url "https://github.com/killme2008/aviatorscript/archive/refs/tags/aviator-5.2.4.tar.gz"
+  sha256 "92b6388ec58da2d987b617aeee204ea54c5a9f0a1d768abdaec9c610555027a5"
+  license "LGPL-3.0-or-later"
+
+  depends_on "maven" => :build
+  depends_on "openjdk"
+
+  def install
+    system "mvn", "package", "-DskipTests"
+    inreplace "bin/aviator", /export AVIATOR_VERSION=".*?"/, "export AVIATOR_VERSION=\"#{version}\""
```

Comment:
What is `AVIATOR_VERSION` set to currently? Also, I don't quite understand the intent of `.*?` here. (I maybe just don't grok regex)